File: IntroToLINQ/IntroToLINQ_CSharp/Student.cs
using System;

namespace IntroToLINQ
{
    public class Student
    {
        public int StudentID { get; set; }
        public string FullName { get; set; }
        public Sex Sex { get; set; }
        public int GCSEGrade { get; set; }

        public Student(string name, int id, Sex sx, int grade)
        {
            StudentID = id;
            FullName = name;
            Sex = sx;
            GCSEGrade = grade;
        }

        public Student()
        {

        }

        public string Summary()
        {
            return StudentID + " " + FullName;
        }

    }

    public enum Sex
    {
        Male,Female
    }


}

File: IntroToLINQ/IntroToLINQ_VB/Student.vb
Public Class Student
    Public Property StudentID As Integer
    Public Property FullName As String
    Public Property Year As YearGroup
    Public Property GCSEGrade As Integer

    Public Sub New(ByVal name As String, ByVal id As Integer, ByVal yg As YearGroup, ByVal grade As Integer)
        StudentID = id
        FullName = name
        Year = yg
        GCSEGrade = grade
    End Sub

    Public Sub New()
    End Sub

    Public Overrides Function ToString() As String
        Return StudentID & " " & FullName
    End Function
End Class

Public Enum YearGroup
    U6
    L6
End Enum
